drivers/i2c/Kconfig.it8xxx2

```diff
@@ -59,6 +59,12 @@ config I2C_CQ_MODE_MAX_PAYLOAD_SIZE
 	  This is the command queue mode payload size which size
 	  up to 2k bytes.
 
+config I2C_ENHANCE_CODE_IN_RAM
```

Comment:
The other Kconfig symbols inside `if I2C_ITE_ENHANCE` don't use the "ENHANCE" keyword.  You can just call this `I2C_CODE_IN_RAM`